app/bundles/CoreBundle/Resources/views/Notification/notification.html.twig

```diff
@@ -1,26 +1,15 @@
-<div class="media pt-sm pb-sm pr-md pl-md nm bdr-b notification" id="notification{{ n.id }}">
-    <span class="pull-left mt-xs" style="width:36px">
-        {% if n.iconClass starts with 'img:' %}
-        <span class="img-wrapper img-rounded">
-            <img class="media-object" src="{{ n.iconClass|slice(4) }}" />
-        </span>
-        {% else %}
-        {% set tooltip = n.type ? ' data-toggle="tooltip" title="{% trans %}mautic.notifications.type.{{ n.type }}{% endtrans %}' : '' %}
-        <i class="ri-2x {{ n.iconClass ? n.iconClass : 'ri-star-line' }}"{{ tooltip }}></i>
-        {% endif %}
-    </span>
-    <a href="javascript:void(0);" class="btn btn-default btn-xs btn-nospin pull-right do-not-close" data-toggle="tooltip" title="{% trans %}mautic.core.notifications.clear{% endtrans %}" onclick="Mautic.clearNotification({{ n.id }});"><i class="ri-close-line do-not-close"></i></a>
-    {% if not n.isRead %}
-        <span class="pull-right is-unread text-danger"><i class="ri-asterisk"></i></span>
-    {% endif %}
+<div class="alert {% if n.type is defined %}alert-{{ n.type }}{% endif %} ma-0 bdr-b notification" id="notification{{ n.id }}">
```

Comment:
For better readability, the following could be considered, I don't know what could be the alter type but this will solve the purpose,
```suggestion
<div class="alert {{ n.type|default('info') }} ma-0 bdr-b notification" id="notification{{ n.id }}">
```

Reply:
We can have 5 different types of notification (non-specified (default), success, info, warning and error/danger - there are docs waiting to be merged for this new notif model)

When the type is non-specified, the final div has only the default `alert` class and is like this:
`<div class="alert ma-0 bdr-b notification" id="notification1">`

![image](https://github.com/user-attachments/assets/eedbf9bc-a942-426f-81f6-784faf727fd2)


When it's info, for example, it generates an extra class `alert-info` this way:
`<div class="alert alert-info ma-0 bdr-b notification" id="notification1">`

![image](https://github.com/user-attachments/assets/0a5bf74d-7809-4299-8df2-f6553ea1d8a7)


Do you think that we should have the info as default instead of the non-specified?